kolibri/core/content/zip_wsgi.py

```diff
@@ -0,0 +1,448 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import codecs
+import json
+import logging
+import mimetypes
```

Comment:
I think this is the only place the mimetype sniffing gets used, so should restore it here.